lib/Bio/EnsEMBL/DataCheck/Checks/CompareMSANames.pm

```diff
@@ -0,0 +1,101 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CompareMSANames;
+
+use warnings;
+use strict;
+use Moose;
+use Test::More;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CompareMSANames',
+  DESCRIPTION    => 'The species_sets from the previous database are still present',
+  GROUPS         => ['compara', 'compara_multiple_alignments'],
+  DATACHECK_TYPE => 'advisory',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['method_link', 'method_link_species_set', 'species_set_header']
+};
+
+sub tests {
+  my ($self) = @_;
+  
+  Bio::EnsEMBL::Registry->load_all($self->registry_file);
+
+  my $curr_dba = $self->dba;
+  my $prev_dba = Bio::EnsEMBL::Compara::DBSQL::DBAdaptor->go_figure_compara_dba('compara_prev');
+  my $curr_db_name = $curr_dba->dbc->dbname;
+  my $prev_db_name = $prev_dba->dbc->dbname;
+  my $curr_mlss_adap = $curr_dba->get_MethodLinkSpeciesSetAdaptor;
+  my $prev_mlss_adap = $prev_dba->get_MethodLinkSpeciesSetAdaptor;
+  my @mlss_types = qw ( PECAN EPO EPO_LOW_COVERAGE CACTUS_HAL );
+  my @curr_mlsses;
+  my @prev_mlsses;
+  
+  foreach my $mlss_type ( @mlss_types ) {
+    my $curr_mlss_ids = $curr_mlss_adap->fetch_all_by_method_link_type($mlss_type);
+    my $prev_mlss_ids = $prev_mlss_adap->fetch_all_by_method_link_type($mlss_type);
+    push @curr_mlsses, @$curr_mlss_ids;
+    push @prev_mlsses, @$prev_mlss_ids;
+  }
+  
+  my $curr_count = scalar(@curr_mlsses);
+  my $prev_count = scalar(@prev_mlsses);
+  
+  my $desc_1 = "$curr_db_name has no fewer mlsses than $prev_db_name";
```

Comment:
No particular convention, just need to try and be clear about the intention of the test (which is often quite difficult to do, while remaining succinct). I agree that it's generally easier to understand a positive statement, because if the test fails, you need to negate that to understand the problem. And that's a bit easier if you're not having to juggle double negatives.